test/ng/directive/selectSpec.js

```diff
@@ -1123,5 +1123,19 @@ describe('select', function() {
       }).toThrowMinErr('ng','badname', 'hasOwnProperty is not a valid "option value" name');
     });
 
+    it('should set providedEmptyOption.selected to false', function() {
+      scope.options = ['a'];
+      scope.model = '';
+      compile(
+          '<select ng-model="model" ng-options="opt for opt in options">' +
+            '<option id="empty_option" value="">Empty Option</option>' +
+          '</select>');
+      var opt = element.find("option")[0];
+      expect(!opt.getAttribute("selected")).toBe(false);
```

Comment:
> why not expect(opt.getAttribute("selected")).toBe(true)

Indeed. And why not `expect(element.find("option").eq(0).attr("option")).toBe("seleced");`, which is what I had before.  That uses a library to ensure cross browser compatibility, which is how it should be done.  As the commit that follows says, I changed it to what it is now because it didn't work in ie9.

The slightly longer version of the story is appears jqLite() has a bug and returns something different under ie9, and so the travis-ci run failed.  That left me with `getAttribute`, which on browsers I tested returns a variety of values (`"selected"`, `true`, `null`, `undefined`). The lowest common denominator is it returns something truthy or falsey, which is what the code tests for.